contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
I'd be glad to take a look if you upload it along your existing libs, but ATM I'm very distracted and busy so it will need to wait for later.